aci/containers.go

```diff
@@ -53,7 +53,7 @@ func (cs *aciContainerService) List(ctx context.Context, all bool) ([]containers
 	if err != nil {
 		return nil, err
 	}
-	var res []containers.Container
+	res := []containers.Container{}
```

Comment:
Unrelated change is unrelated :D

Reply:
It is related.

The previous results in `nil` in the case of an empty list. The new results in an `empty slice`.

This is related to the `JSON` serialization. It handles differently `nil` and an `empty slice`.

Reply:
Maybe `make([]containers.Cotainer, 0)` could make that more specific.